lldb/include/lldb/Target/Language.h

```diff
@@ -95,21 +95,24 @@ class Language : public PluginInterface {
   class EitherTypeScavenger : public TypeScavenger {
   public:
     EitherTypeScavenger() : TypeScavenger() {
-      for (std::shared_ptr<TypeScavenger> scavenger : { std::shared_ptr<TypeScavenger>(new ScavengerTypes())... }) {
+      for (std::shared_ptr<TypeScavenger> scavenger :
+           {std::shared_ptr<TypeScavenger>(new ScavengerTypes())...}) {
```

Comment:
Can this use `make_shared`?

Reply:
FWIW, these changes were not introduced by @junior-jl, they're probably the result of running `clang-format` on the whole file.

Reply:
Yes, that was indeed the case. I believe I removed all the unrelated modifications from the PR in this last push.